### PR TITLE
Batch Size of Four

### LINKED ISSUE #11: Batch Size of Four
**Describe the enhancement**
Set the batch size of 4. A smaller batch size like 4 could help improve the speed of convergence and accuracy of the model.

For this modification to work, you must adjust the correct calculation of the model output in the `train_model.py` file starting at line 226: 
```
voutputs = model(vinputs)
pred_probab = torch.max(voutputs.data, 1)[1]
vloss = loss_fn(voutputs, vlabels)
running_test_loss += vloss
correct += (
    (pred_probab == vlabels)
    .type(torch.float)
    .sum()
    .item()
)
```

## Resource
- https://stackoverflow.com/questions/35050753/how-big-should-batch-size-and-number-of-epochs-be-when-fitting-a-model

### PR DESCRIPTION
**Pull request by sagedemage**
Set the batch size to 4. Changed the method of the correct calculation of the model output; this improved the accuracy of the model.

### CODE REVIEW

**Review by sagedemage — COMMENTED** [review]
LGTM!
